.depcheckrc.yml

```diff
@@ -58,6 +58,22 @@ ignores:
   - 'resolve-url-loader'
   # jest environments
   - 'jest-environment-jsdom'
+  # webpack
+  - '@pmmmwh/react-refresh-webpack-plugin' # dev tool
```

Comment:
Why are there so many new items needed here?

Reply:
These are used in a webpack configuration. Since `depcheck` is a static analyzer it can't detect that they are used in the webpack config.